MODULE.bazel

```diff
@@ -1,6 +1,137 @@
-###############################################################################
-# Bazel now uses Bzlmod by default to manage external dependencies.
-# Please consider migrating your external dependencies from WORKSPACE to MODULE.bazel.
-#
-# For more details, please check https://github.com/bazelbuild/bazel/issues/18958
-###############################################################################
+module(name = "rules_twirl")
+
+bazel_dep(name = "bazel_skylib", version = "1.7.1")
+bazel_dep(name = "buildifier_prebuilt", version = "7.3.1")
+bazel_dep(name = "rules_java", version = "7.11.1")
+bazel_dep(name = "rules_pkg", version = "1.0.1")
+bazel_dep(name = "rules_jvm_external", version = "6.4")
+bazel_dep(name = "rules_scala_annex")
+
+rules_scala_annex_version = "bzlmod-migration"
+
+archive_override(
+    module_name = "rules_scala_annex",
+    integrity = "sha256-Gt+S2R53KPhfsK66bhOSXowjb35Ouf24zcT3xFZcyAs=",
+    strip_prefix = "rules_scala-{}".format(rules_scala_annex_version),
+    urls = ["https://github.com/lucidsoftware/rules_scala/archive/refs/heads/{}.zip".format(rules_scala_annex_version)],
+)
+
+bazel_dep(name = "stardoc", version = "0.7.1")
+
+register_toolchains(
+    "//:repository_default_toolchain_21_definition",
+    "//scala:zinc_2_13",
+    "//scala:zinc_3",
+    "//twirl-toolchain:twirl-2-13",
+    "//twirl-toolchain:twirl-3",
+)
+
+# Please ensure these stay up-to-date with the versions in `versions.bzl`. Unfortunately,
+# `MODULE.bazel` files can't call `load`, so we have to copy them here.
+scala_2_13_version = "2.13.14"
+
+scala_3_version = "3.5.1"
+
+twirl_version = "2.0.7"
+
+protobuf_version = "4.28.3"
+
+scopt_version = "4.1.0"
+
+specs2_version = "4.20.8"
+
+zinc_version = "1.10.1"
```

Comment:
Scope creep, but can we please update this to 1.10.4? Annex is already on that version, so it should be as simple as changing the version and updating the lockfile.

Reply:
Sure thing.